docs/API.md

```diff
@@ -20,33 +20,43 @@ gulp.src('client/templates/*.jade')
   .pipe(gulp.dest('build/minified_templates'));
 ```
 
-`glob` refers to [node-glob syntax](https://github.com/isaacs/node-glob) or it can be a direct file path.
-
 #### globs
 Type: `String` or `Array`
 
-Glob or array of globs to read.
+Glob or array of globs to read. Globs use [node-glob syntax](https://github.com/isaacs/node-glob) except that negation is fully supported.
+
+A glob that begins with `!` excludes matching files from the glob results up to that point. For example, consider this directory structure:
+
+    client/
+      a.js
+      bob.js
+      bad.js
+
+The following expression matches `a.js` and `bad.js`:
+
+    gulp.src(['client/*.js', '!client/b*.js', 'client/bad.js'])
+    
 
 #### options
 Type: `Object`
 
 Options to pass to [node-glob] through [glob-stream].
 
-gulp adds some additional options in addition to the [options supported by node-glob][node-glob documentation] and [glob-stream]:
+gulp supports all [options supported by node-glob][node-glob documentation] and [glob-stream] except `ignore` and adds the following options.
```

Comment:
A colon precedes a block or list within a section. Text preceding subsections does not use a colon, since the scope of what the colon applies to would not be clear.

Examples from HTML5 spec: [List](http://www.w3.org/TR/html5/introduction.html#extensibility), [Subsections (microsyntaxes)](http://www.w3.org/TR/html5/infrastructure.html#common-microsyntaxes), [Subsections (collections)](http://www.w3.org/TR/html5/infrastructure.html#collections), [List and subsection (kind of content)](http://www.w3.org/TR/html5/dom.html#kinds-of-content)